src/components/SnapshotImage.tsx

```diff
@@ -65,10 +65,10 @@ export const Container = styled.div<{ href?: string; target?: string }>(
 );
 
 interface SnapshotImageProps {
-  componentName: Test["story"]["component"]["name"];
-  storyName: Test["story"]["name"];
+  componentName?: NonNullable<NonNullable<Test["story"]>["component"]>["name"];
```

Comment:
@ghengeveld a lot of our API types are nullable when maybe they shouldn't be? I'm not sure if you have religion about this but it does make things a bit of a pain.

Reply:
The general thought is that fields that require a db lookup (or any async operation) should be nullable. However, since a test with no story/component info is rather useless, and there is no way to delete such data from the db (as would be possible for e.g. a user or a comment), I think it would be okay to mark `story` and `component` non nullable. I'm happy if you make that change.

Reply:
I'll follow up with some non-null tweaks to our API.